source/configuring-ember/configuring-your-app.md

```diff
@@ -0,0 +1,32 @@
+Ember-CLI ships with support for managing your application's environment. Ember-CLI will build an environment config file at `config/environment`. Here, you can define an ENV object for each environment (development, test and production). For now, this is limited to the three environments mentioned.
+
+The ENV object contains two important keys: 1) EmberENV, and 2) APP. The first can be used to define Ember feature flags (see the [Feature Flags guide](http://emberjs.com/guides/configuring-ember/feature-flags/)). The second can be used to pass flags/options to your application instance.
+
+You can access these environment variables in your application code by importing from `../config/environment` or `your-application-name/config/environment`.
```

Comment:
Like https://github.com/emberjs/guides/pull/520/files#r36064293 I think this is something we should agree on the style and stick to it, and explain it once somewhere.